src/cli.ts

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+import 'reflect-metadata'
```

Comment:
Add comment for what we import globally `reflect-metadata`, as a helper for DI system :)

Reply:
@jotamusik did you add the comment?